### ISSUE TITLE
v1.2.1335-1 import csv from Files panel not showing data preview

### ISSUE THREAD

**Issue by pachamaltese** [rstudio/rstudio#4563]
Hi
I'm using Ubuntu 18 build and reader 1.3.1

When I try to import a file from the file panel the code preview shows:
```
library(readr)
dataset <- read_csv(NULL)
View(dataset)
```

in a 2nd try it returns:
```
library(readr)
file <- read_csv("~/Downloads/file.csv")
View(file)
```
and from the 2nd try it shows the data preview.

It doesn't happen with v1.1

**Comment by ronblum** [comment]
@pachamaltese Thank you for reporting this issue. We'll review as we're developing RStudio.

I'm able to reproduce this with a `.xlsx` file using RStudio Desktop 1.2.1335 on MacOS 10.14. It only happens the first time a file is used to import data in an R session. Any subsequent attempts in the same session—whether clicking on "Updating" or hitting "Cancel" and trying to import again—works correctly. However, when the R session is restarted, the issue recurs.

**Comment by pachamaltese** [comment]
yes !!!

the same happens to me !!!!

—————

*Mauricio Vargas Sepúlveda 帕夏*
Do you like Data Science? visit pacha.hk
你爱科学数据专吗？你走pacha.hk

**Comment by ronblum** [comment]
Verified fixed in RStudio Desktop 1.2.1522-1 on MacOS 10.14.6.
